main.go

```diff
@@ -36,6 +37,17 @@ import (
 	"github.com/kubernetes-incubator/external-dns/source"
 )
 
+type logOutputSplitter struct{}
+
+// Splits log output, error and fatal to stderr and the rest to stdout
+func (splitter *logOutputSplitter) Write(p []byte) (n int, err error) {
+	if bytes.Contains(p, []byte("level=debug")) || bytes.Contains(p, []byte("level=info")) ||
```

Comment:
Yes, I agree. It is hacky. However I'm not really sure how to solve it in a nicer way as the only thing we can act on is the log line. Looking at the references it seems like they are not intending to fix it in `logrus` (especially not now since it is, according to the README, looking for a maintainer).